### ISSUE TITLE
AI planes spawn intersecting mid-air at Khalkhalah

### ISSUE THREAD

**Issue by SnappyComebacks** [dcs-liberation/dcs_liberation#518]
When doing the "Golan Heights Battle" campaign (Winter Modern, Player: USA 1990, Enemy: Syria 1982), planes spawn midair intersecting with each other at Khalkhalah.  Several enemy planes crash within a few seconds of starting the mission.  This happens within multiple missions.

There are three planes in this photo.
![Overlapping_clean](https://user-images.githubusercontent.com/74509817/101289053-415ba500-37b7-11eb-92b4-ef24242acfdc.png)

**Comment by DanAlbert** [comment]
Already fixed in the dev build. If it's still present it's a map bug that we can't fix.

**Comment by MedievalBunny** [comment]
I'm having this problem as well on the current build of 2.3.2

It appears so though aircraft are attempting to spawn on the helicopter pads starting with H01, but the spawn waypoint is set to Turning Point and the aircraft end up spawning in on top of each other. 

It may well be a map issue in DCS, when I set the spawn type to Ramp Start they jump to a completely different airbase. I'm on Stable as well, so this may be fixed on Open Beta. But I figured I'd offer some more info in case it's helpful. I attached a .miz where the problem is occurring. 
[liberation_nextturn_crash.zip](https://github.com/Khopa/dcs_liberation/files/5726356/liberation_nextturn_crash.zip)
